controllers/machinedeletionremediation_controller.go

```diff
@@ -27,7 +29,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"strings"
 
 	"github.com/medik8s/machine-deletion/api/v1alpha1"
```

Comment:
Do we plan to change the repo name after the PR?
`github.com/medik8s/machine-deletion` -> `github.com/medik8s/machine-deletion-remediation`

Reply:
yes, absolutely. I wanted something working with the current path, then change it